src/pages/Contribution.pug

```diff
@@ -93,16 +94,16 @@ block article
     +image(7, imgpath, "Freezer and Fridge Box Organization", 60)
     
     :markdown-it(html)
-        - A further step towards an organized lab is using a table in which you keep track of all your orders and planned orders. This way, every member can check what was ordered and not just the person responsible for orders.
+        A further step towards an organized laboratory is using **a table to keep track of all the orders and planned orders**. This way, every member can check what was ordered and not just the person responsible for orders.
         
     - var imgpath = "" + require("../assets/img/contribution/Orders.jpg")["default"]
     +image(8, imgpath, "Laboratory Orders Organization")
     
     :markdown-it(html)
-        - Labeling of Tubes. We cannot stress this enough, please label your tubes properly to avoid any confusion and mistakes in the lab. Always write down what is in the tube, the date, and the concentration (if applicable). Since the labels wash off easily when frozen or they get in touch with any detergent, you should use circular paper stickers for labelling (see image below). If you have multiple copies or aliquotes of one reagent, you can mark the tube in usage with an ‘X’ so that you empty one tube before using another one.
+        **Tubes should bei labbeled properly!** We cannot stress this enough, please label the tubes properly to avoid any confusion and mistakes in the lab. Always write down what the tube contains, the date, and the concentration (if applicable). Since the labels wash off easily when frozen or they get in touch with any detergent, circular paper stickers for labelling (see image below) should be used. If multiple copies or aliquotes of one reagent exists, the tube in usage can be marked with an ‘X’ so that one tube is emptied before using another one.
```

Comment:
```suggestion
        **Tubes should bei labbeled properly!** 
        We cannot stress this enough, please label the tubes properly to avoid any confusion and mistakes in the lab. 
        Always write down what the tube contains, the date, and the concentration (if applicable). 
        Since the labels wash off easily when frozen or they get in touch with any detergent, circular paper stickers for labelling (see image below) should be used. 
        If multiple copies or aliquotes of one reagent exists, the tube in usage can be marked with an ‘X’ so that one tube is emptied before using another one.
```